views/locations/locations_delete.php

```diff
@@ -46,11 +46,17 @@
         if ($success) {
             $deleteRes = $db->query("DELETE FROM facilitatorclassattendance WHERE sitename = $1", [$sitename]);
             $success = $deleteRes && pg_result_error_field($deleteRes, PGSQL_DIAG_SQLSTATE) == 0;
+        }
 
```

Comment:
Should also delete from ClassOffering, which would be necessary if  `hasRole(Role::Superuser)` and `$notConnected` is false